### ISSUE TITLE
nightlights-hd-3bands item indexing question

### ISSUE THREAD

**Issue by anayeaye** [NASA-IMPACT/cloud-optimized-data-pipelines#136]
## What
There seems to be an indexing problem for the `nightlights-hd-3bands` items--I noticed that the collection `summaries` object is empty and when I tried to compute it manually I found that I couldn't select any items for this collection id in postgres. This is super strange because the simple get items search **does** work https://staging-stac.delta-backend.xyz/collections/nightlights-hd-3bands/items.

## Notes
I can find the collection by id but can't find items by collection_id or the collection property in the "contents" (which is not the way to search but I am having trouble selecting the 3 band nightlights items in postgres). 
```sql
select * from collections where id = 'nightlights-hd-3bands'; # 1 collection
select * from items where collection_id = 'nightlights-hd-3bands'; # no results
select * from items where "content"->>'collection' = 'nightlights-hd-3bands'; # no results
```
## AC
Items indexed and `update_default_summaries(nightlights-hd-3bands)` produces a collection summary.

**Comment by anayeaye** [comment]
@abarciauskas-bgse @slesaad @xhagrg It turns out this is happening for multiple collections. I did a quick staging database comparison of all collection ids to the distinct list of collection_ids in the items table ([here's a spreadsheet](https://docs.google.com/spreadsheets/d/1nJ7FhoGL22PTCOLPuRukxVYueOM_HmhbC50FzQkTOIM/edit?usp=sharing)). I there might be a couple collection records we want to delete but I don't know why others are not represented in the items table. Can you all have a look and let me know what you think?

Two things seem possible
- Items were directly edited in the database instead of using pypgstac which also handles some indexing logic behind the scenes
- Maybe the ingest version of pypgstac deployed is not compatible with the version of pypgstac currently installed in the staging database? There are big enough changes in the pypgstac released with v0.5 that we should coordinate on the upgrade.

Collection ids not in the items table
```
blue-tarp-planetscope
grdi-v2-raster
HLSS30
IS2SITMOGR4
nightlights-hd-3bands
OMI_trno2
OMSO2PCA
OMSO2PCA_COG
```

**Comment by anayeaye** [comment]
I was able to run the summary function for all collections this AM. Issue resolved. Thanks!
